Resources/Prototypes/Voice/speech_emotes.yml

```diff
@@ -336,6 +336,13 @@
   icon: Interface/Emotes/weh.png
   chatMessages: [Wehs!]
 
+- type: emote
+  id: Hew
+  name: chat-emote-name-hew
+  category: Vocal
+  icon: Interface/Emotes/hew.png
+  chatMessages: [Hews!]
```

Comment:
since you are here, could you make locale support to it and to weh above it

Reply:
Added localization support, tested, works ingame.